src/Microsoft.ML.Data/Transforms/CopyColumnsTransform.cs

```diff
@@ -130,18 +109,21 @@ public sealed class Arguments : TransformInputBase
             public Column[] Column;
         }
 
-        // Factory method for SignatureLoadDataTransform.
-        private static IDataView Create(IHostEnvironment env, ModelLoadContext ctx, IDataView input)
+        /// <summary>
+        /// Factory method to create from arguments
+        /// </summary>
+        public static IDataTransform Create(IHostEnvironment env, Arguments args, IDataView input)
```

Comment:
>public [](start = 8, length = 6)

Why it become public?

Reply:
This remained public, please see the code in master:
[https://github.com/dotnet/machinelearning/blob/20761a3575eee232798f962868d4fd714334ccac/src/Microsoft.ML.Data/Transforms/CopyColumnsTransform.cs#L167](https://github.com/dotnet/machinelearning/blob/20761a3575eee232798f962868d4fd714334ccac/src/Microsoft.ML.Data/Transforms/CopyColumnsTransform.cs#L167)

Its the diff that makes it look like it transitioned from private to public.

---
In reply to: [222523784](https://github.com/dotnet/machinelearning/pull/1141#discussion_r222523784) [](ancestors = 222523784)